src/Microsoft.Data/PrimitiveColumnComputations.cs

```diff
@@ -0,0 +1,1864 @@
+﻿
+
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+// Generated from PrimitiveColumnComputations.tt. Do not modify directly
+
+using System;
+
+namespace Microsoft.Data
+{
+    internal interface IPrimitiveColumnComputation<T>
+        where T : struct
+    {
+        void Abs(PrimitiveColumnContainer<T> column);
+        void All(PrimitiveColumnContainer<T> column, out bool ret);
+        void Any(PrimitiveColumnContainer<T> column, out bool ret);
+        void Cummax(PrimitiveColumnContainer<T> column);
```

Comment:
I think we should spell out these words - or at least make them more ".NET"-ish.

`CumulativeMax` would be a better name IMO.